### ISSUE TITLE
Improve logging

### ISSUE THREAD

**Issue by tran-tien-dat** [TEAMMATES/teammates#8608]
- **Environment**: everywhere 

**Description of feature/enhancement**
Logs currently contain HTML for formatting. Also, the log-related classes have a few bugs which should be fixed as well.

**Justification**

HTML shouldn't be present in the log messages as we should have a clear separation between content and presentation. Parts of the formatting of the log for displaying is already handled by the application, so leaving HTML inside the log only reduces cohesion. By moving formatting totally to the application, developers will find it easier to read the logs as well as the logic of the log-related classes. The application will also be more secure.

I am working on a PR to fix this.
**Existing similar features and their shortcomings**

**Comment by wkurniawan07** [comment]
@tran-tien-dat please organize this issue's description. Do not be hasty when having an idea/thought.
- What do you mean by environment "everywhere"? Environment is not sub-systems/features of the system.
- What is hoped to be fixed here? The HTML for formatting, or the "few bugs"? The latter should be (1) more precisely qualified/quantified and (2) left to other issue(s). You may find #6502 relevant.
- Can provide some examples/screenshots for context?

**Comment by tran-tien-dat** [comment]
Thanks @wkurniawan07 for the constructive feedback. I have edited the PR accordingly. Hope it's better now.

**Comment by xpdavid** [comment]
![image](https://user-images.githubusercontent.com/13977235/37096700-6d5436c0-2254-11e8-8a2c-d0ae2c55fcc1.png)

@tran-tien-dat I guess this is what u mean right? should be `getActionResponse` rather than `getActionName` as only response can have `ACTION_RESULT_FAILURE` or `ACTION_RESULT_SYSTEM_ERROR_REPORT`

In your corresponding PR, u are also addressing the Sanitization issue. Is it also relevant to this issue?

**Comment by tran-tien-dat** [comment]
>  I guess this is what u mean right? should be getActionResponse rather than getActionName as only responses can have ACTION_RESULT_FAILURE or ACTION_RESULT_SYSTEM_ERROR_REPORT

Yeah but the 2nd function is not meaningfully used anywhere so I didn't change it. Should it be marked for removal, together with everything associated with `ACTION_RESULT_SYSTEM_ERROR_REPORT`?

**Comment by xpdavid** [comment]
Can just remove it or u can create a first-timer issue regarding that (better).
